package.json

```diff
@@ -17,22 +17,30 @@
   "author": "Patrick Leary <patrick.r.leary@gmail.com>",
   "license": "MIT",
   "dependencies": {
+    "@hapi/joi": "^15.1.0",
+    "@jagql/framework": "*",
     "async": "^2.6.1",
     "bluebird": "^3.5.2",
     "body-parser": "^1.18.3",
     "cacheman": "^2.2.1",
     "cacheman-file": "^0.2.1",
     "carto": "^1.2.0",
     "compression": "^1.7.3",
+    "debug": "*",
     "ejs": "^2.6.1",
     "elasticmaps": "github:inaturalist/elasticmaps",
     "elasticsearch": "^15.5.0",
     "es6-promise-pool": "^2.5.0",
     "express": "^4.16.4",
+    "express-graphql": "*",
+    "express-openapi": "^4.6.5",
     "fast-csv": "^2.4.1",
     "form-data": "^2.3.3",
     "geoip-lite": "^1.3.4",
+    "graphql": "*",
+    "graphql-fields": "*",
     "handlebars": "^4.5.3",
+    "hapi-joi-to-swagger": "github:pleary/hapi-joi-to-swagger#6c17eccd5a6f6e3b0a50646ad83a6459c11f8809",
```

Comment:
shall I move this over to be owned by inaturalist?

Reply:
Yes please.